### ISSUE TITLE
samples day 4 today 5 

### ISSUE THREAD

**Issue by olieter** [Anders-Holst/xled_plus#3]
Raise HTTPError(http_error_msg, response=self) requests.exceptions.HTTPError: 404 Client Error: Not Found for url:

**Comment by Anders-Holst** [comment]
Hi @olieter 
Thanks for your report. However, it is truncated. It would have been useful to see the entire stack trace that was presumably written, where it would say at what call at what line in the code it happened.

But let me guess: The difference with day 4 and day 5 from previous examples is that they use the layout. The layout is not available on the old firmware that the leds are shipped with. When I realized this, I updated (together with the day 4 effect) the "highcontrol" code to check for this (there is some uncertainty though with which firmware version exactly it becomes available, but I hope I got it right).

So I suppose: You have never upgraded the firmware on your leds, and you downloaded xled_plus some time before December 4? Try either updating the firmware, or updating xled_plus, and with some luck the problem will go away. (Updating the firmware, and afterwards mapping the layout again, will give the best effect. Otherwise the gradient will follow the string, not the physical layout.)

By the way, before you upgrade, what firmware version are you using now? 2.2.1 or something else? Just wanna check if my test on the version is enough.
